Scripts/Train_Model.py

```diff
@@ -29,16 +29,7 @@
 
 
 def main():
-    """
-    Trains a using the ww_model.py class
-
-    Args:
-        None
-
-    Returns:
-        None
-
-    """
+    # Trains a model using the ww_model.py class
```

Comment:
Is there a reason we got rid of the args and return labels?

Reply:
This wasn't done anywhere else so it was inconsistent, if we want to reintroduce this we ought to do this for every single method.